docker-compose.yml

```diff
@@ -0,0 +1,29 @@
+# Copyright 2018 Bitwise IO, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+version: "3.6"
+
+volumes:
+  cargo-registry:
+  cargo-git:
+
+services:
+  sawtooth-pbft:
+    image: sawtooth-pbft:$ISOLATION_ID
+    volumes:
+      - .:/project/sawtooth-pbft
+      - cargo-registry:/root/.cargo/registry
+      - cargo-git:/root/.cargo/git
+    build: .
+    command: cargo test
```

Comment:
If unit tests are preferred to be run this way, then it might be a good idea to remove the file `tests/pbft_unit_tests.yaml` so there isn't duplication

Reply:
Sure, sounds good 👍